src/QLExpression.h

```diff
@@ -50,31 +50,35 @@ struct IExpression {
 	/**
 	 * Return the name of the index which, if it exists, indexes by the values of this expression
 	 */
-	virtual Standalone<StringRef> get_index_key() const { return StringRef(); }
+	virtual std::string get_index_key() const { return {}; }
 };
 
+std::string encodeMaybeDotted(std::string fieldname);
```

Comment:
Is this defined anywhere? It might possibly conflict with the one defined in `ExtUtil`

Reply:
Its the same one as in `ExtUtil`. There seems to be some kind of circular dependency in header files. Saving that cleanup for later and took a shortcut by just adding declaration here.

Reply:
OK